### PR TITLE
Created Book class

### LINKED ISSUE #1: Create a Book Class
Create a Book class that has:
- [x] A constructor with the parameters of isbn, title, and price.

Properties:
- [x] The books ISBN
- [x] The books Title
- [x] The books retail price

Methods:
- [x] An overriding ToString method with concatenates the ISBN, Title, and Price Together

### PR DESCRIPTION
**Pull request by TungKimProgrammer**
Created Book class that has:
- Constructor
- Properties: ISBN, Title, Price
- Method: ToString() 

Closes #1

### CODE REVIEW

File: LibraryBookRegistration/Book.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LibraryBookRegistration
+{
+    /// <summary>
+    /// Represents a customer
```
Comment (crex424) [inline LibraryBookRegistration/Book.cs:10]: Documentation for Book at line 10: I believe should be "Represents a book" not "Represents a customer".

Comment (TungKimProgrammer) [reply LibraryBookRegistration/Book.cs:10]: Thank you very much!

**Comment by crex424** [comment]
Looks good.

**Comment by crex424** [comment]
Shall I merge it to main?

**Comment by TungKimProgrammer** [comment]
I can do it. It's a good time for me to gain some experience.

**Comment by crex424** [comment]
Roger that.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Pull request merged**